explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -225,31 +225,35 @@ object AladinCell extends ModelOptics with AladinCommon:
       .withHooks[Props]
       .useContext(AppContext.ctx)
       // Request guide star candidates if obsTime changes more than a month or the base moves
-      .useEffectKeepResultWithDepsBy((p, _) => (p.obsTime, p.asterism.baseTracking)):
-        (props, ctx) =>
-          (obsTime, baseTracking) =>
-            import ctx.given
+      .useEffectResultWithDepsBy((props, _) =>
+        (props.siderealDiscretizedObsTime, props.asterism.baseTracking)
+      ): (props, ctx) =>
+        (siderealDiscretizedObsTime, baseTracking) =>
+          import ctx.given
 
-            if (props.needsAGS)
-              (for
-                _          <- props.obsConf
-                                .flatMap(_.agsState)
-                                .foldMap(_.async.set(AgsState.LoadingCandidates))
-                candidates <- CatalogClient[IO]
-                                .requestSingle:
-                                  CatalogMessage.GSRequest(baseTracking, obsTime)
-              yield candidates)
-                .guarantee:
-                  props.obsConf.flatMap(_.agsState).foldMap(_.async.set(AgsState.Idle))
-            else none.pure
+          if (props.needsAGS)
+            (for
+              _          <- props.obsConf
```

Comment:
So we run ags only if time changes more thaan a month, except for average pa
that seems right